Improve SEO metadata across all docs pages (#397)
powersync-ja/powersync-docs: client-sdks/orms/overview.mdx

---
title: "ORM Support Overview"
sidebarTitle: "ORM Support"

> description: "Choose a type-safe ORM or query builder for PowerSync instead of raw SQL, with options for JavaScript, Kotlin, Flutter, and Swift SDKs."

Reviewer: ```suggestion
description: "Choose a type-safe ORM or query builder for PowerSync instead of writing raw SQL queries."
```
